tests.js

```diff
@@ -5095,6 +5178,150 @@ window.Specs = {
 		}
 	},
 
+	"predefined-counter-styles": {
+		"title": "Ready-made Counter Styles",
+		"links": {
+			"tr": "predefined-counter-styles",
+			"dev": "predefined-counter-styles"
```

Comment:
The draft of this note is directly hosted on GitHub.
```suggestion
			"dev": "predefined-counter-styles",
			"devtype": "github"
```

Reply:
I fix it.